docs/scaffolding.md

```diff
@@ -46,9 +46,9 @@ Your skelebot is ready to go!
 ### Results
 Scaffolding will always create the skelebot.yaml file, which is what defines the projects as a Skelebot project.
 
-If scaffolding is run without the '--existing' flag, it will also create several folders for storing data, models, notebooks,  queries, output, and code. It will also generate placeholder yaml files for five different environments as well as an example bash job that can be executed through Skelebot.
+If scaffolding is run without the `--existing` flag, it will also create several folders for storing data, models, notebooks,  queries, output, and code.
 
-The structure produced is merely a suggestion and does not need to be adhered to in order for Skelebot to function properly. The only thing Skelebot actually needs is the skelebot.yaml file. Once you have this file in your project, you can execute the help command, 'skelebot --help' and get started using it right away.
+The structure produced is merely a suggestion and does not need to be adhered to in order for Skelebot to function properly. The only thing Skelebot actually needs is the skelebot.yaml file and the VERSION file, although the VERSION file. Once you have these file in your project, you can execute the help command, `skelebot --help` and get started using it right away.
```

Comment:
`although the VERSION file.` I think something's missing here :)

Also: `these files`.